ui/basicTab/basicTab.html

```diff
@@ -148,36 +148,5 @@
         <input name="attr_featCP" class="featCP" value="" type="number" />
       </fieldset>
     </div>
-
-    &emsp; &emsp;
-    <!-- weight penalty table -->
-    <div class="flex-stack">
-      <div class="center wideish bold">Weight Penalties</div>
```

Comment:
Weight penalties are now redundant with the header.
We can remove this in favor of opening up more space for the tab content.